docs/services/gcp-services-guide.md

```diff
@@ -16,10 +16,8 @@ indent: true
      2. [Crossplane Steps](#crossplane-steps)
```

Comment:
Really like the cleaned up flow for the Azure Service Guide. Suggest following that here in the GCP Service Guide as much as possible.

Suggest the following ToC:

1. Pre-requisites
1. Preparation
1. Set Up Crossplane
    1. Install
    1. Cloud Provider
    1. Resource Classes
    1. Configure Managed Service Access (note: you only need one private connection for all services)
1. Provision MySQL
   1. Resource Claim
1. Install Wordpress
1. Clean Up
1. Conclusion and Next Steps